src/cmake/config/linux-gcc.txt

```diff
@@ -66,6 +66,7 @@ set (c_compiler_flags_common
     -Wno-sign-compare
     -Wno-unused-function
     -Wno-reorder
+    -Wno-strict-aliasing
```

Comment:
Enabling it produces a number of  "dereferencing type-punned pointer will break strict-aliasing rules" warnings that we'd need to fix in a future PR.
For now, this fixes the release build and we didn't have this warning enabled before in any case.